cooperatives.yml

```diff
@@ -253,3 +253,11 @@ lyseontech:
   url: http://LT.coop.br/
   description: "Lyseon Tech - Open source technology solutions"
   location: Brazil
+
+fiqus:
```

Comment:
here you should enter `bagimsizatolye` or a similar internal id.

Reply:
Ah fixed it. I guess I was a little excited :-)

Reply:
haha np :)